db/post_migrate/20221214133921_rename_setting_in_category_custom_field.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class RenameSettingInCategoryCustomField < ActiveRecord::Migration[6.1]
+  def up
+    DB.exec(<<~SQL)
+      UPDATE category_custom_fields
+      SET name = 'create_as_post_voting_default'
+      WHERE name = 'create_as_qa_default'
+    SQL
+  end
+
+  def down
+    DB.exec(<<~SQL)
```

Comment:
I believe we don't usually do this and raise an exception instead. However, It's simple enough, so I don't see why not 👍